.github/workflows/samples-scala.yaml

```diff
@@ -32,6 +32,7 @@ jobs:
           - samples/server/petstore/scalatra
           - samples/server/petstore/scala-finch # cannot be tested with jdk11
           - samples/server/petstore/scala-http4s-server
+          - samples/server/petstore/scala/cask
```

Comment:
should it be `samples/server/petstore/scala-cask` instead?

Reply:
yes - good catch. updated